### PR TITLE
Feat add filter by salary

### PR DESCRIPTION
**Pull request by mr-fox93**
image

### CODE REVIEW

File: src/app/(profile)/profile.helpers.ts [hunk 1]
```diff
@@ -46,11 +46,13 @@ export const createFiltersObjFromSearchParams = (
     [JobOfferFiltersEnum.availability]: [],
     [JobOfferFiltersEnum.location]: [],
     [JobOfferFiltersEnum.search]: [],
+    [JobOfferFiltersEnum.salary]: [],
   }
 
   for (const [key, value] of searchParams.entries()) {
     filters[key as keyof SearchParamsFilters] = value.split(',')
   }
+  console.log(filters)
```
Comment (Sumick) [inline src/app/(profile)/profile.helpers.ts:55]: **nit:** Log to remove

File: src/app/(profile)/(components)/ProfileList/filters.ts [hunk 2]
```diff
@@ -62,3 +62,21 @@ export const filterByFullName = (searchTermFilter: string) => {
       .includes(searchTermFilter.toUpperCase())
   }
 }
+
+export const filterBySalary =
```
Comment (Sumick) [inline src/app/(profile)/(components)/ProfileList/filters.ts:66]: **nit:** Let's move all filter functions to `profile.helpers.ts` file

**Review by Sumick — APPROVED** [review]
Looks good! Before merging let's just move all filters to different file and remove log

**Review by mmcode9407 — APPROVED** [review]
GJ 👍

**Pull request merged**